src/lib/getKeyDownHandlers.ts

```diff
@@ -7,17 +8,24 @@ import {
 } from 'src/lib/accessibility/types'
 import { State, PropsWithVarsAndStyles } from '../themes/types'
 
+const rtlKeyMap = {
+  [keyboardKey.ArrowRight]: keyboardKey.ArrowLeft,
+  [keyboardKey.ArrowLeft]: keyboardKey.ArrowRight,
+}
+
 /**
  * Assigns onKeyDown handler to the Component's part element, based on Component's actions
  * and keys mappings defined in Accessibility behavior
  * @param {AccessibilityActionHandlers} componentActionHandlers Actions handlers defined in a component.
  * @param {KeyActions} behaviorKeyActions Mappings of actions and keys defined in Accessibility behavior.
  * @param {State & PropsWithVarsAndStyles} props The props which are used to invoke onKeyDown handler passed from top.
+ * @param {boolean} isRtl Indicates if Left and Right arrow keys should be swapped in RTL mode.
```

Comment:
Would recommend using the `rtl` name, the same name that is used in the Theme, or `isRtlEnabled` if you want to use the verb :)

Reply:
would stick for `isRtlEnabled`, thx